spec/index.md

```diff
@@ -14,6 +14,15 @@ lang: en
 
 This is version 1.0.0 of the specification.
 
+- [Terminology](#terminology)
+- [Introduction](#introduction)
+- [micro:bit Power Modes](#micro:bit-power-modes)
```

Comment:
This link doesn't seem to work.

Reply:
Fixed in https://github.com/microbit-foundation/spec-power-management/pull/4/commits/536de62afe0990a39ee90e5f8ee963f389883415 Worked in local preview but not on GitHub